### ISSUE TITLE
Feature request: auto select default account on startup

### ISSUE THREAD

**Issue by ssb22** [thundernest/k-9#1620]
I have two accounts, one of which I want to access 99.9% of the time and the other I want to access 0.1% of the time (haven't measured but that's what it feels like).  It would be nice if I didn't have to go through the account picker when starting K-9 Mail.  A "go straight to default account" option would be good.  Then I can go back up through "Folders" to "Accounts" in the rare occasions when I want to check the other account.

Thanks so much.

**Comment by philipwhiuk** [comment]
You can add a widget that does this. If the existing widgets aren't suitable could you clarify why not. I think a widget is the probably right way to address this but I may be missing something.

**Comment by ssb22** [comment]
Thanks.  The "K-9 Unread" widget (I've only found this one) can indeed be set to choose an account by default, and it also obeys that account's "Auto-expand folder" setting (at least it does when multiple accounts are present; for some reason it didn't when I tried it with only one account in the database).

The problem with widgets though is they take more taps to reach.  I use an app called "Simple Home" for big fonts on the home screen (I'm sight impaired), and Simple Home doesn't "do" widgets, so I have to install the widget as a "small app" and reach it via Android 4.4's app list (and then it floats on top of K9 and I have to explicitly close it once the folder is on-screen).  It's not half as convenient as being able to go straight to my chosen folder in my chosen account when K9 is launched from Simple Home.  (And I'd really rather not change back to the phone's default "Xperia™ Home" instead of Simple Home.)
